src/renderer/modals/FullNode/steps/StepAccounts.js

```diff
@@ -0,0 +1,133 @@
+// @flow
+import React, { useCallback, useState } from "react";
+import Box from "~/renderer/components/Box";
+import Text from "~/renderer/components/Text";
+import { useSelector } from "react-redux";
+import { accountsSelector } from "~/renderer/reducers/accounts";
+import { getAccountCurrency } from "@ledgerhq/live-common/lib/account";
+import { Trans } from "react-i18next";
+import Button from "~/renderer/components/Button";
+import CurrencyBadge from "~/renderer/components/CurrencyBadge";
+import FormattedVal from "~/renderer/components/FormattedVal";
+import CryptoCurrencyIcon from "~/renderer/components/CryptoCurrencyIcon";
+import { getCryptoCurrencyById } from "@ledgerhq/live-common/lib/currencies";
+import Input from "~/renderer/components/Input";
+import InfoCircle from "~/renderer/icons/InfoCircle";
+import ToolTip from "~/renderer/components/Tooltip";
+import type { FullNodeSteps } from "~/renderer/modals/FullNode";
+import styled from "styled-components";
+
+const Row = styled(Box).attrs(() => ({
+  horizontal: true,
+  alignItems: "center",
+  bg: "palette.background.default",
+  px: 2,
+  mb: 2,
+  flow: 3,
+}))`
+  height: 40px;
+  border-radius: 4px;
+`;
+
+const Accounts = () => {
+  // FIXME Not using the AccountList componenent because styles differ quite a bit, we should unify.
+  const accounts = useSelector(accountsSelector);
+  const [numberOfAccountsToScan, setNumberOfAccountsToScan] = useState(10);
+  const currency = getCryptoCurrencyById("bitcoin");
+  const bitcoinAccounts = accounts.filter(a => getAccountCurrency(a) === currency);
+
+  const onUpdateNumberOfAccountsToScan = useCallback(value => {
+    const newNumberOfAccounts = parseInt(value) || 1;
```

Comment:
please give 10 to second parameter of parseInt.
I suggest to fallback to 1 as soon as the value is `isNaN()`, `!isFinite()` or `< 1`.
and also to actively deny an edit that would make it higher than 1000 as you seem to do before => maybe just use "maxlength 3" on the input?